.github/workflows/xcode_check.yml

```diff
@@ -0,0 +1,24 @@
+name: Check xcode completeness
+
+on:
+  push:
+    paths:
+    - 'source/**'
+    - 'tests/**'
```

Comment:
Should this check really run on changes to `tests/`? Wouldn't changes to the Xcode project itself be more apt?

Reply:
Good point, changed to check only updates to the script itself.